scripts/qa_crowbarsetup.sh

```diff
@@ -1335,7 +1335,7 @@ function create_repos_yml()
             additional_repos+=" SLES12-SP1-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12-SP1:/x86_64/update/"
             # FIXME: enable when Cloud 7 test updates are available
             # additional_repos+=" SUSE-OpenStack-Cloud-7-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/OpenStack-Cloud:/7:/x86_64/update/"
-            [ $hacloud == 1 ] && additional_repos+=" SLE12-SP1-HA-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12-SP1:/x86_64/update/"
+            [[ $hacloud == 1 ]] && additional_repos+=" SLE12-SP1-HA-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12-SP1:/x86_64/update/"
```

Comment:
AFAIK the original comparison is correct in bash (tested in 4.3.46) [1]:

```bash
a=10
[ $a == 10 ] && echo 'GOOD'    # shows GOOD
[ $a == 1 ] && echo 'BAD'    # shows nothing
```

but if the double bracket is more correct: +1

[1] http://tldp.org/LDP/abs/html/comparison-ops.html